### PR TITLE
Add documentation for using PRIMO with Binder

### PR DESCRIPTION
**Pull request by apd-ypuranik**
## Fixes/Addresses/Summary/Motivation:

Includes documentation for how to use PRIMO on Binder: a free, open-source and public environment where PRIMO can be utilized with no installation necessary

## Changes proposed in this PR:
- Updated documentation

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Review by radhakrishnatg — APPROVED** [review]

**Review by ruonanli61 — APPROVED** [review]

File: docs/binder.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+machine. However, we recognize that not every user has access to the required IT permissions on their machines to
+carry out the installation. For such users, we have made PRIMO available via a free and public service through Binder.
+This allows anyone to access, experiment with and run PRIMO from a web browser with **no installation** needed.
+The steps to access PRIMO via Binder follow. Please note that:
+
+* The Binder environment is **public and therefore insecure**. Sensitive data should NOT be uploaded/used on this environment. This environment should be used only for **testing, exploration and experimentation** purposes.
+* The Binder environment is automatically deleted after 10 minutes of inactivity. To avoid lost work, please download your work locally on a periodic basis. 
+  
+Please reach out to the NETL team (primo@netl.doe.gov) for any questions or concerns. 
+We remain committed to helping states implement MERP in any way we can.
```
Comment (strahlw) [inline docs/binder.rst:14]: should be implement **"the"** MERP

**Review by strahlw — APPROVED** [review]
One very minor comment. How does binder work with versioning? Does it build an environment from a .yml file or similar?

File: docs/binder.rst (hunk 1 above)
Comment (apd-ypuranik) [reply docs/binder.rst:14]: Thank you for the attention to detail. Appreciate it :-)

**Comment by apd-ypuranik** [comment]
> One very minor comment. How does binder work with versioning? Does it build an environment from a .yml file or similar?

@strahlw : Binder checks for a few different locations when looking for config. It looks for a .binder folder, or an environment.yml file or requirements.txt. You are able to use any tag/branch/commit to create a Binder environment, the badge added on our project homepage will always point to the latest version by default.

**Pull request merged**